### PR TITLE
fix: Set plugin metadata

### PR DESCRIPTION
**Pull request by hermanschaaf**
This sets plugin metadata on all the CloudQuery-maintained plugins, following an update to plugin-sdk v4.16.0, which requires this metadata to be present for the `package` command to succeed.

### CODE REVIEW

File: plugins/source/firestore/resources/plugin/plugin.go [hunk 1]
```diff
@@ -5,12 +5,19 @@ import (
 	"github.com/cloudquery/plugin-sdk/v4/plugin"
 )
 
-var version = "development"
```
Comment (hermanschaaf) [inline plugins/source/firestore/resources/plugin/plugin.go:8]: The fact that `version` was unexported here was actually a bug, as even official versions of the plugin right now report their version to be `development`:

```
./plugin --version
firestore version development
```

**Comment by disq** [comment]
Couple of questions:

- Why `var` when they are constants?
- Why not use strings inside plugin init and define them as vars (or consts) when it's only in one place anyway?

**Review by disq — APPROVED** [review]

**Comment by hermanschaaf** [comment]
@disq Keeping them var because we may want to replace these values with `ldflag` in the future (like we already do for version), and keeping them grouped because I think it's nice to have this metadata visible/editable in one place, rather than in a function call.

**Comment by hermanschaaf** [comment]
Closing for now, I did a bad merge

**Pull request closed without merging**